src/redux/missions/missionsSlice.js

```diff
@@ -0,0 +1,105 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+/**
+ * The URL of the SpaceX missions API.
+ * @constant {string}
+ */
+const MISSION_API_URL = 'https://api.spacexdata.com/v3/missions';
+
+/**
+ * An asynchronous Redux thunk action creator to fetch missions from the SpaceX API.
+ *
+ * @type {AsyncThunk<Array, void, {}>}
+ */
+const fetchMissions = createAsyncThunk('missions/fetchMissions', async () => {
+  const response = await axios.get(MISSION_API_URL);
+  return response.data;
```

Comment:
- [ ] Kindly filter the data from the missions API too. It is not all the information from the API that should be dispatched to the store.

  ![image](https://github.com/lily4178993/space-traveller/assets/54848363/358a9c29-06df-4513-8b09-ad87e3012427)